domains/dheeran.localplayer.dev.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Personal website that's hosted on github pages.",
+  "domain": "localplayer.dev",
+  "subdomain": "dheeran",
+  "owner": {
+    "repo": "https://github.com/teamblackwing/dheeran",
+    "email": "dheerandps@gmail.com"
+  },
+  "record": {
+    "CNAME": "teamblackwing.github.io"
```

Comment:
Your Vercel linked site works, however this is on GitHub. Your GitHub previews do not work. Make sure you have this properly set. Re-open this pull request when finished.